imap_processing/cdf/config/imap_idex_l1_variable_attrs.yaml

```diff
@@ -0,0 +1,420 @@
+int_fillval: &int_fillval -9223372036854775808
+min_epoch: &min_epoch -315575942816000000
+max_epoch: &max_epoch 3155630469184000000
+
+data_min: &data_min 0
+data_max: &data_max 4096
+
+sample_rate_min: &sample_rate_min -130
+sample_rate_max: &sample_rate_max 130
+
+int_maxval: &int_maxval 9223372036854775807 # np.iinfo(np.int64).max
+
+# l1_data_base
+l1_data_base: &l1_data_base # ScienceAttrs
+  VALIDMIN: *min_epoch
+  VALIDMAX: *max_epoch
+  DISPLAY_TYPE: spectrogram
+  DEPEND_0: epoch
+  FORMAT: ""
+  UNITS: dN
+  VAR_TYPE: data
+  # VAR_PURPOSE: PRIMARY
+  CATDESC: ""
+  FIELDNAM: ""
+  FILLVAL: *int_fillval
+  LABLAXIS: ""
+
+l1_tof_base: &l1_tof_base
+  <<: *l1_data_base
+  DEPEND_1: time_high_sr
+
+l1_target_base: &l1_target_base
+  <<: *l1_data_base
+  DEPEND_1: time_low_sr
+
+epoch:
+  <<:  *l1_data_base
+  CATDESC: Time, number of nanoseconds since J2000 with leap seconds included
+  FIELDNAM: epoch
+  FILLVAL: *int_fillval # GlobalConstants.INT_FILLVAL,
+  LABLAXIS: epoch
+  FORMAT: ""  # Supposedly not required, fails in xarray_to_cdf
+  UNITS: ns
+  VALIDMIN: *min_epoch # GlobalConstants.MIN_EPOCH
+  VALIDMAX: *max_epoch # GlobalConstants.MAX_EPOCH
+  VAR_TYPE: support_data
+  SCALETYP: linear
+  MONOTON: INCREASE
+  TIME_BASE: J2000
+  TIME_SCALE: Terrestrial Time
+  REFERENCE_POSITION: Rotating Earth Geoid
+
+sample_rate_base: &sample_rate_base # FloatAttrs
+  DISPLAY_TYPE: no_plot
+  CATDESC: ""
+  FIELDNAM: ""
+  FILLVAL: -1.0e+31 # DOUBLE FILL VAL
+  VALIDMIN: *sample_rate_min
+  VALIDMAX: *sample_rate_max
+  DEPEND_0: epoch
+  FORMAT: F64.5
+  LABLAXIS: Time
+  UNITS: microseconds
+  VAR_TYPE: support_data # Not in attrs file
+  VAR_NOTES:  The number of microseconds since the event.
+              0 is the start of data collection, negative
+              numbers represent data collected prior to a dust event
+
+trigger_base: &trigger_base # ScienceAttrs
+  VALIDMIN: 0
+  VALIDMAX: *int_maxval
+  DISPLAY_0: epoch
+  DISPLAY_TYPE: no_plot
+  CATDESC: ""
+  FIELDNAM: ""
+  VAR_TYPE: support_data
+  FILLVAL: *int_fillval
+  LABLAXIS: ""
+  FORMAT: I12
+  UNITS: ""
+
+# TODO: All of these have var_notes in idex_cef_attrs. Should I add these?
```

Comment:
Looks like `VAR_NOTES` is an optional attribute in terms of ISTP compliance, and just offers additional information. If it is not too much trouble to add them, I think it is worth doing.

Reply:
And it looks like a few other instruments use them (`hi` and `swe`)